docs/designers-developers/developers/themes/theme-support.md

```diff
@@ -47,6 +47,14 @@ add_action( 'after_setup_theme', 'mytheme_setup_theme_supported_features' );
 
 ## Opt-in features
 
+## Default block styles
```

Comment:
This is one of the most important aspects of Gutenberg theme support, so it needs to be more visible/higher in the docs.

Reply:
We should probably expand a bit here with a line or two to clarify its importance.